### ISSUE TITLE
Verify return flow rate calculations during sizing when multiple return nodes are used in a single zone

### ISSUE THREAD

**Issue by rraustad** [NREL/EnergyPlus#6955]
Issue overview
--------------
Follow up to #6726

The IF/ELSE IF where you made this change says "Make no return air flow adjustments during sizing" for the IF and requires numRetNodes == 1. So if there are 2 return nodes then the code falls to the ELSE IF and doesn't set returnNodeMassFlow when doing sizing? Doesn't returnNodeMassFlow need a value during sizing if there are 2 return nodes?

             // Return node 1 is special
             if (returnNum == 1) {
                 // Make no return air flow adjustments during sizing
                 if ((DataGlobals::DoingSizing || DataGlobals::isPulseZoneSizing) && numRetNodes == 1) {
                     returnNodeMassFlow = ExpTotalReturnMassFlow;
                     if (airLoop > 0) {
                         if (!DataAirSystems::PrimaryAirSystem(airLoop).OASysExists) {
                             ExpTotalReturnMassFlow = max(0.0, ExpTotalReturnMassFlow - thisZoneEquip.ZoneExhBalanced + thisZoneEquip.ZoneExh);
                             returnNodeMassFlow = ExpTotalReturnMassFlow;
                         }
                     }
                } else if (!DataGlobals::DoingSizing && !DataGlobals::isPulseZoneSizing) {
                    if (thisZoneEquip.NumReturnFlowBasisNodes > 0) {
                        // Set base return air flow rate for node 1 using basis node flow rates
                        Real64 basisNodesMassFlow = 0.0;
                        for (int nodeNum = 1; nodeNum <= thisZoneEquip.NumReturnFlowBasisNodes; ++nodeNum) {
                            returnNodeMassFlow += DataLoopNode::Node(thisZoneEquip.ReturnFlowBasisNode(nodeNum)).MassFlowRate;
                            basisNodesMassFlow += DataLoopNode::Node(thisZoneEquip.ReturnFlowBasisNode(nodeNum)).MassFlowRate;
                        }
                        returnNodeMassFlow = max(0.0, (returnNodeMassFlow * returnSchedFrac));
                        returnNodeMassFlow = max(0.0, (basisNodesMassFlow * returnSchedFrac));
                        fixedReturn(returnNum) = true;
                    } else {
                        // If only 1 return node, use the standard return mass flow
                         if (numRetNodes == 1) {
                             returnNodeMassFlow = max(0.0, (ExpTotalReturnMassFlow * returnSchedFrac * airLoopReturnFrac));
                         }
                     }
                 }
             }
             totReturnFlow += returnNodeMassFlow;
             retNodeData.MassFlowRate = returnNodeMassFlow;

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
Yeah, funny thing, I'm fairly certain this block from above is dead code:
```
                     if (!DataAirSystems::PrimaryAirSystem(airLoop).OASysExists) {
                         ExpTotalReturnMassFlow = max(0.0, ExpTotalReturnMassFlow - thisZoneEquip.ZoneExhBalanced + thisZoneEquip.ZoneExh);
                         returnNodeMassFlow = ExpTotalReturnMassFlow;
                     }
```
`airLoop` will never be > 0 when `DoingSizing` is true. At least a quick debug run of 5ZoneAirCooled shows that it never gets here. Maybe there's a special case that does, but I think this block belongs up before the sizing check.

**Comment by rraustad** [comment]
My thought is that during sizing the return node mass flow and temp/humrat are needed to calculate a mixed air temperature. If return mass flow is always 0 then what is sizing using for mixed air temp? I certainly don't see OA conditions during coil sizing so not sure what is happening here.
